README.md

```diff
@@ -1,17 +1,18 @@
-# Equinix Metal Virtual Routing and Forwarding (VRF)
+# Equinix Metal Virtual Routing and Forwarding (VRF) 
 
 [![Experimental](https://img.shields.io/badge/Stability-Experimental-red.svg)](https://github.com/equinix-labs/standards#about-uniform-standards)
 [![terraform](https://github.com/equinix-labs/terraform-equinix-metal-vrf/actions/workflows/integration.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-metal-vrf/actions/workflows/integration.yaml)
 
-# Attention! VRF is currently only supported with Dedicated Metal Ports
-
 # VRF deployment on Equinix Platform
 
 Metal Virtual Routing & Forwarding (VRF) is a layer 3 networking service. It's implemented in Metal's networking devices (routers, switches etc) with built-in HA function. This Terraform script provides VRF deployments on Equinix Metal platform where a Metal Gateway, a VRF and a number of metal nodes are deployed. The metal VRF is connected to a pair of customer colo edge devices via a pair of redundant Virtual Connections (VC) created in a redundant dedicated fabric port (see high-level diagram below). The VRF is used to establish BGP sessions with colo network devices (or network edge devices) and advertise the specified network IPs to the devices.
 
-<img width="1223" alt="image" src="https://user-images.githubusercontent.com/46980377/203126361-e9030734-3905-45f7-acbb-71df0ba928ad.png">
+![Metal-VRF-Github-0928-2023](https://github.com/equinix-labs/terraform-equinix-metal-vrf/assets/46980377/f3f2718c-bb53-4744-b1f1-e5f4a0116017)
```

Comment:
I don't know how sticky this image URL will be. Let's assume it is persistent.
It currently redirects me to https://github-production-user-asset-6210df.s3.amazonaws.com/46980377/271362897-f3f2718c-bb53-4744-b1f1-e5f4a0116017.png which seems like a new distribution approach compared to the previous URL: https://user-images.githubusercontent.com/46980377/203126361-e9030734-3905-45f7-acbb-71df0ba928ad.png